policyengine_us/parameters/gov/states/nj/tax/income/deductions/property_tax/limit.yaml

```
@@ -2,11 +2,13 @@ description: New Jersey allows for a property tax credit or deduction on up to t
 metadata:
   unit: currency-USD
   period: year
-  label: New Jersey property tax deduction/credit threshold
+  label: New Jersey property tax deduction/credit limit
```

To distinguish from the limit of the deduction/credit value
```suggestion
  label: New Jersey property tax deduction/credit property tax limit
```